perf/data/sentences.js

```diff
@@ -16,9 +16,9 @@ const sentences = [
   'I will never be this young again. Ever. Oh damn… I just got older.',
   'Wow, does that work?',
   'Malls are great places to shop; I can find everything I need under one roof.',
-  'She borrowed the book from him many years ago and hasn\'t yet returned it.',
+  "She borrowed the book from him many years ago and hasn't yet returned it.",
   'A glittering gem is not enough.',
-  'She wrote him a long letter, but he didn\'t read it.',
+  "She wrote him a long letter, but he didn't read it.",
```

Comment:
prettier uses the "avoid-escape" approach to quotes but google's eslint config uses the "always single quotes" approach. so i guess we need to turn one of the two off (or reconfigure it). i don't mind which but either eslintrc or prettierrc has to change so they line up